.github/workflows/version-bump.yml

```diff
@@ -0,0 +1,140 @@
+name: 'Version Bump'
+on:
+  workflow_dispatch:
+    inputs:
+      release_line:
+        description: 'Release Line'
+        required: true
+        default: next
+        type: choice
+        options:
+        - next
+        - main
+  
+  # Run Weekly on Wednesday at 10:00 UTC time
+  schedule:
+    - cron: "0 10 * * 3"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  RELEASE_NAME: ''
+  HAS_CHANGES: ''
+  CURRENT_VERSION: ''
+  RELEASE_VERSION: ''
+
+jobs:
+  version-bump:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout backstage'
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+
+      # Beginning of yarn setup
+      - name: use node.js 14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+          registry-url: https://registry.npmjs.org/ # Needed for auth
+      - name: cache all node_modules
+        id: cache-modules
+        uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-node_modules-${{ hashFiles('yarn.lock', '**/package.json') }}
+      - name: find location of global yarn cache
+        id: yarn-cache
+        if: steps.cache-modules.outputs.cache-hit != 'true'
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - name: cache global yarn cache
+        uses: actions/cache@v2
+        if: steps.cache-modules.outputs.cache-hit != 'true'
+        with:
+          path: ${{ steps.yarn-cache.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+      - name: yarn install
+        run: yarn install --frozen-lockfile
+      # End of yarn setup
+
+      - name: 'Set release name'
+        shell: pwsh
```

Comment:
Powershell is going to be tricky for us to debug, and evolve, we are very much in the bash camp so it would be great if we can convert these powershell sections to bash. 
`curl` and [jq](https://stedolan.github.io/jq/) will be essential to fetch contents and parse json.

An alternative if you don't feel comfortable with bash is to do a node script separately or use `actions/github-script@v6` to be able to use nodejs. Thinking about it a node script might actually be the best place to start for this as it's much easier to read and test.

Reply:
Alright, I've refactored this to move away from PowerShell and prefer node for larger scripts or bash for simple things 👍